lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3450,28 +3438,22 @@ class PropertyEmitter : public EmitterBase,
     assert(state.pp.getListener() == &state.saver);
   }
 
-  /// Emit the specified value as an SVA property or sequence with an enable
-  /// signal. This is the entry point to print an entire tree of property or
-  /// sequence expressions in one go.
-  void emitEnabledProperty(
-      Value property, Value enable,
-      PropertyPrecedence parenthesizeIfLooserThan = PropertyPrecedence::Lowest);
+  void emitAssertPropertyBody(
+      Value property, Value disable,
+      PropertyPrecedence parenthesizeIfLooserThan = PropertyPrecedence::Lowest,
+      bool ignoreNonEmptyTokens = false);
```

Comment:
The boolean flag is a bit rough here I think, and it's basically "I'm calling this as the tail of a different entry-point-y function" (since it does the flush) and is for internal use only as a result.

Additionally, functionally that boolean is dead in non-assert builds.

Putting the common functionality in a helper method the entry-ish methods (the public ones) call to implement their behavior is the usual pattern and avoids the need to answer questions about the flag.  This would mean the entry-points do the buffer management dance.

Thanks!